README.md

```diff
@@ -176,6 +179,31 @@ stringData:
 type: Opaque
 ```
 
+#### Migrating data from an old AWX instance
+
+To migrate data from the postgresql pod of an AWX deployment in Openshift for Kubernetes, the Custom Resource needs to know about the connection details. Those connection details should be stored as a secret and either specified as `tower_old_postgres_configuration_secret` at the CR spec level, or simply be present on the namespace under the name `<resourcename>-old-postgres-configuration`.
```

Comment:
> To migrate data from the postgresql pod of an AWX deployment in Openshift for Kubernetes

Is there a typo here? Cant really tell what this was trying to say.

Maybe something like: "To migrate data from an older AWX installation,"